centreon/tests/e2e/features/Dashboards/10-widget-metric-graph-configuration/index.ts

```diff
@@ -0,0 +1,395 @@
+  cy.getByTestId({ testId: 'confirm' }).click();
+});
+
+Then("the Metrics Graph widget is added to the dashboard's layout", () => {
+  cy.getByTestId({ testId: 'warning-line-200-tooltip' }).should('exist');
+  cy.getByTestId({ testId: 'critical-line-400-tooltip' }).should('exist');
+});
+
+Then('the information about the selected metric is displayed', () => {
+  cy.get('[data-testid^="warning-line-"][data-testid$="-tooltip"]').should(
+    'exist'
+  );
+  cy.getByTestId({ testId: 'Min' }).should('exist');
+  cy.getByTestId({ testId: 'Max' }).should('exist');
+  cy.getByTestId({ testId: 'Avg' }).should('exist');
+});
+
+Given('a dashboard featuring having Metrics Graph widget', () => {
+  cy.insertDashboardWithMetricsGraphWidget(
+    dashboards.default,
+    metricsGraphWidget
+  );
+  cy.visit(`${Cypress.config().baseUrl}/centreon/home/dashboards`);
+  cy.wait('@listAllDashboards');
+  cy.getByLabel({
+    label: 'view',
+    tag: 'button'
+  })
+    .contains(dashboards.default.name)
+    .click();
+  cy.getByLabel({
+    label: 'Edit dashboard',
+    tag: 'button'
+  }).click();
+  cy.wait('@performanceData');
+  cy.getByTestId({ testId: 'MoreVertIcon' }).click();
+  cy.getByLabel({
+    label: 'Edit widget',
+    tag: 'li'
+  }).realClick();
+});
+
+When(
+  'the dashboard administrator user updates the custom warning threshold',
+  () => {
+    cy.get('[class^="MuiAccordionDetails-root"]').eq(1).scrollIntoView();
+    cy.contains('Custom').find('input').eq(0).click();
+    cy.getByLabel({
+      label: 'Thresholds',
+      tag: 'input'
+    }).type('500');
+  }
+);
+
+Then(
+  'the Metrics Graph widget is refreshed to display the updated warning threshold horizontal bar',
+  () => {
+    cy.getByTestId({ testId: 'warning-line-500-tooltip' }).should('exist');
+  }
+);
+
+When(
+  'the dashboard administrator user updates the custom critical threshold',
+  () => {
+    cy.get('input[type="radio"][value="custom"]').eq(1).click({ force: true });
+    cy.getByLabel({
+      label: 'Thresholds',
+      tag: 'input'
+    })
+      .eq(1)
+      .type('300', { force: true });
+  }
+);
+
+Then(
+  'the Metrics Graph widget is refreshed to display the updated critical threshold horizontal bar',
+  () => {
+    cy.getByTestId({ testId: 'critical-line-300-tooltip' }).should('exist');
+  }
+);
+
+When(
+  'the dashboard administrator user updates a threshold to a value beyond the default range of the Y-axis',
+  () => {
+    cy.get('input[type="radio"][value="custom"]').eq(1).click({ force: true });
+    cy.getByLabel({
+      label: 'Thresholds',
+      tag: 'input'
+    })
+      .eq(1)
+      .clear();
+    cy.getByLabel({
+      label: 'Thresholds',
+      tag: 'input'
+    })
+      .eq(1)
+      .type('600', { force: true });
+  }
+);
+
+Then(
+  'the Y-axis of the Metrics Graph widget is updated to reflect the change in threshold',
+  () => {
+    cy.getByTestId({ testId: 'critical-line-600-tooltip' }).should('exist');
```

Comment:
as we've seen this is kind of hard to check the values of the X/Y-axes on the graph via Cypress; should we still keep the test as-is according to you?

Reply:
I can't understand your need, could you please give me more details